dev-docs/test-3rd-party-apps-manually.md

```diff
@@ -67,32 +44,8 @@ The commands should be available at `https://github.com/GoogleCloudPlatform/ops-
 
 Take Nginx for example:
 ```
-sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
-logging:
-  receivers:
-    nginx_access:
-      type: nginx_access
-    nginx_error:
-      type: nginx_error
-  service:
-    pipelines:
-      nginx:
-        receivers:
-          - nginx_access
-          - nginx_error
-metrics:
-  receivers:
-    nginx_metrics:
-      type: nginx
-      stub_status_url: http://127.0.0.1:80/nginx_status
-      collection_interval: 30s
-  service:
-    pipelines:
-      nginx:
-        receivers:
-          - nginx_metrics
-EOF
-sudo service google-cloud-ops-agent restart
+curl https://raw.githubusercontent.com/GoogleCloudPlatform/ops-agent/master/integration_test/third_party_apps_data/applications/nginx/enable > enable.sh
```

Comment:
[Optional] `enable-nginx-telemetry.sh`?

Reply:
That would require more manual work when copy pasting this command to rename the file when testing one app. I'd defer to the developer whether they want to rename it.